src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,443 @@
+	"ModeOfAdministration": "bolus",
+	"MolarActivity": 353.51,
+	"MolarActivityUnit": "GBq/umol",
+	"MolarActivityMeasTime": "13:04:42",
+	"TimeZero": "13:04:42",
+	"ScanStart": 0,
+	"InjectionStart": 0,
+	"FrameTimesStart": [0, 10, 20, 30, 40, 50, 60, 80, 100, 120, 140, 160, 180, 240, 300, 360, 420, 480, 540, 660, 780, 900, 1020, 1140, 1260, 1380, 1500, 1800, 2100, 2400, 2700, 3000, 3300, 3600, 3900, 4200, 4500, 4800, 5100, 5400, 5700, 6000, 6300, 6600, 6900],
+	"FrameDuration": [10, 10, 10, 10, 10, 10, 20, 20, 20, 20, 20, 20, 60, 60, 60, 60, 60, 60, 120, 120, 120, 120, 120, 120, 120, 120, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300],
+	"AcquisitionMode": "list mode",
+	"ImageDecayCorrected": true,
+	"ImageDecayCorrectionTime": 0,
+	"ReconMethodName": "3D-OSEM-PSF",
+	"ReconMethodParameterLabels": ["subsets","iterations"],
+	"ReconMethodParameterUnit": ["none","none"],
+	"ReconMethodParameterValues": [16,10],
+	"ReconFilterType": "none",
+	"AttenuationCorrection": "[137Cs]transmission scan-based"
+}
+```
+
+## Blood recording data
+
+This section includes all data needed to perform blood analysis for PET data.
+The section may be omitted completely if blood measurements of radioactivity were not made.
+
+Template:
+
+```Text
+sub-<participant_label>/
+ 	[ses-<session_label>/]
+	pet/
+	sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_recording-<label>_blood.tsv
+	sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_recording-<label>_blood.json
+
+```
+
+Blood data belongs in the /pet folder along with the corresponding PET data.
+However, the blood data also follows the [inheritance principle](../02-common-principles.md#the-inheritance-principle) and may be moved to an upper level folder
+if it does not change, for example, with multiple reconstructions.
+The blood data is most often recorded using an autosampler for continuous blood samples,
+or manually for discrete blood samples.
+Therefore, the recording key (`recording-<label>`) has two reserved values:
+`autosampler`,
+for continuous blood data measurements;
+and `manual`, for discrete blood data.
+All blood measurements should be reported according to a single time-scale
+in relation to time zero defined by the PET data (Figure 1).
+All definitions used below are in accordance with Innis et al. 2007 ([doi:10.1038/sj.jcbfm.9600493](https://doi.org/10.1038/sj.jcbfm.9600493)).
+
+## Blood sidecar JSON (`*recording-(manual|autosampler)_blood.json`)
+
+If blood measurements were made a `*recording-<label>_blood.json` file is expected that contains
+information about which blood data (plasma, whole blood, and metabolites) is available as well as some blood measurement details.
+These are detailed in the next sections. Different fields are customary depending on if manual or automatic blood sampling is carried out.
```

Comment:
I recommend that for the json (which accompanies the tsv), a link is made to the general section https://bids-specification.readthedocs.io/en/bep-009/02-common-principles.html#tabular-files    

That also specifies how columns in the TSV file are to be documented in the JSON file. 

Another example of that is https://bids-specification.readthedocs.io/en/bep-009/03-modality-agnostic-files.html#phenotypic-and-assessment-data. I wonder whether the JSON fields here should also be represented under `MeasurementToolMetadata` for consistency, or whether they are to be represented at the top level.